packages/react-dom/src/client/ReactDOMHostConfig.js

```diff
@@ -394,6 +399,41 @@ export function removeChildFromContainer(
   }
 }
 
+export function hideInstance(instance: Instance): void {
+  // TODO: Does this work for all element types? What about MathML? Should we
+  // pass host context to this method?
+  instance = ((instance: any): HTMLElement);
+  if (instance.style !== undefined && instance.style !== null) {
+    instance.style.display = 'none';
```

Comment:
What happens if it already has a style attribute that disagrees? What if you hide it first, but then it updates itself with `block` as the value? Or does sequencing guarantee `hideInstance` always runs last?

Reply:
During the commit that hides them, `hideInstance` always comes last because children effects fire before parent effects. While they're timed out, the children will never update. It's a more extreme version of `hidden`.

Reply:
This makes me remember [this](https://github.com/NE-SmallTown/Thinking-in-react/tree/master/packages/react-suspense-polyfill) and [this](https://twitter.com/Heaven_xz/status/972139052748062720) :)